src/compiler/create-parse-config-host.ts

```diff
@@ -11,6 +11,14 @@ export default function createParseConfigHost(
     return input.getFileSystemEntries(path);
   }
 
+  function realpath(path: string): string  {
+    try {
+      return input.realpath(path) || path;
```

Comment:
Is the `|| path` really needed in addition to the `try/catch`? There is no explanation of the failure scenario here.